tests/devapps/WebApi/Controllers/S2SAndOboController.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Cache.CacheImpl;
+using Microsoft.Identity.Client.Internal.Logger;
+using WebApi.Misc;
+using WebApi.MockHttp;
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class S2SAndOboController : ControllerBase
+    {
+        private const string FileCachePath = @"c:\temp\cache";
```

Comment:
Path.GetTempPath() can be used here instead of hard coded

Reply:
This is true, but the point of this sample is to look at the this directory and the files withing, and this simpler.